carma_wm_ctrl/include/carma_wm_ctrl/WMBroadcaster.h

```diff
@@ -268,6 +261,9 @@ class WMBroadcaster
    * It should not be incremented for updates that do not require a full map publication.
    */
   size_t current_map_version_ = 0; 
+  ros::V_string combineParticipantsToVehicle(const ros::V_string& input_participants) const;
```

Comment:
This function was moved but its method comments were deleted. Please add the comments back. It also looks like it was moved from public to private. Please confirm this does not cause build issues